build/Dockerfile

```diff
@@ -16,29 +16,25 @@ ENV PATH=${PATH}:/usr/local/go/bin
 
 # Build WMCB
 WORKDIR /build/
-RUN git clone --single-branch --branch release-4.6 https://github.com/openshift/windows-machine-config-bootstrapper.git
-WORKDIR windows-machine-config-bootstrapper
+COPY windows-machine-config-bootstrapper .
 RUN make build
 
 # Build hybrid-overlay
 WORKDIR /build/
-RUN git clone --single-branch --branch release-4.6 https://github.com/openshift/ovn-kubernetes/
-WORKDIR ovn-kubernetes/go-controller/
+COPY ovn-kubernetes .
+WORKDIR go-controller/
 RUN make windows
 
 # Build Kubernetes node binaries
 WORKDIR /build/
-RUN git clone --branch release-4.6 https://github.com/openshift/kubernetes
-WORKDIR /build/kubernetes
```

Comment:
How is `operator-sdk build` working without this?

Reply:
I do not understand that, since the WORKDIR is /build/ the makefile was accessible